### DISCUSSION TITLE
一直卡在 Starting DoChat /dɑɑˈtʃæt/ ...  ubuntu 16.04

### DISCUSSION THREAD

**Discussion by ZhangDewei** [huan/docker-wechat#97] [General]
🚀 Pulling the docker image done.
🚀 Starting DoChat /dɑɑˈtʃæt/ ...

请问是否操作有误.

**Comment by bxueaa** [comment]
# curl -sL https://raw.githubusercontent.com/huan/docker-wechat/master/dochat.sh | bash

       ____         ____ _           _
      |  _ \  ___  / ___| |__   __ _| |_
      | | | |/ _ \| |   | '_ \ / _` | __|
      | |_| | (_) | |___| | | | (_| | |_
      |____/ \___/ \____|_| |_|\__,_|\__|

      https://github.com/huan/docker-wechat

                +--------------+
               /|             /|
              / |            / |
             *--+-----------*  |
             |  |           |  |
             |  |   盒装    |  |
             |  |   微信    |  |
             |  +-----------+--+
             | /            | /
             |/             |/
             *--------------*

      DoChat /dɑɑˈtʃæt/ (Docker-weChat) is:

      📦 a Docker image
      🤐 for running PC Windows WeChat
      💻 on your Linux desktop
      💖 by one-line of command

🚀 Pulling the docker image...

2.7.1.85: Pulling from zixia/wechat
93956c6f8d9e: Pull complete 
46bddb84d1c5: Pull complete 
15fa85048576: Pull complete 
8aa40341c4fa: Pull complete 
1e85f9d3a0e2: Pull complete 
1e7ba6fed8ef: Pull complete 
522c0a74c4cc: Pull complete 
bb6c04ebadfd: Pull complete 
2d5b3070ed76: Pull complete 
375ed383c7a8: Pull complete 
4e20fedae17a: Pull complete 
517d52506e35: Pull complete 
be64ac073317: Pull complete 
Digest: sha256:e6e9d21c7cd1dfae0484e697f12f5f3c401de2f02e771d061868740e0d26549d
Status: Downloaded newer image for zixia/wechat:2.7.1.85
docker.io/zixia/wechat:2.7.1.85

🚀 Pulling the docker image done.
🚀 Starting DoChat /dɑɑˈtʃæt/ ...

I have the same problem. 
ubuntu 18.04

**Comment by ArtisticZhao** [comment]
你是不是多个显示器? 看看这个FAQ
[No main window after start up with 2 or more monitor setup
](https://github.com/huan/docker-wechat#no-main-window-after-start-up-with-2-or-more-monitor-setup)

**Comment by YalandHong** [comment]
你的问题描述很不详细。

Possible dup of #128
